articles/introduction-to-deep-learning/index.md

```diff
@@ -0,0 +1,123 @@
+
+6. Deep Learning is an essential part of Machine Learning. It provides solutions to many problems in image recognition, speech recognition, and NLP. Deep Learning performs well with images, text, sound recognition as compared to Machine Learning.
+
+### How Deep Learning Works
+The functioning of the brain influences deep Learning algorithms. Neural Networks are made up of layers and nodes, as well as the brain comprises of neurons.
+
+Neural Networks consist of:
+- Input layer
+- Hidden layers
+- Output layer
+
+The input layer receives input data. Hidden layers perform mathematical computations on the information. The output layer gives the result. The problem is solved based on computation from the distributive [weights of all layers](https://www.modev.com/blog/how-deep-learning-works).
+
+A good example to use is the recognition of a house image. The first layer might focus on borderlines. Another layer focus on the object colors, and another the measurements of the image.
+
+Deep Learning systems need a powerful software. This is because they process huge amounts of data and complex computations.
+
+### Why we need Deep Learning
+Deep Learning is revolutionizing so many fields to solve data problems.
+Deep Learning is an excellent and powerful tool. It helps many organizations and developers create exciting AI applications.
+Think of successful surgeries, causing little or no deaths due to surgical errors. Think of a world with no accident due to smart cars. In several years to come, the world will be on another level due to Deep Learning.
+
+### Applications of Deep Learning
+There is massive excitement about artificial intelligence and its subsets. Here are Deep Learning applications that will govern the world.
+
+1. Self-driven cars
+
+Companies such as Google are building driver assistance services. Also, they are teaching computers how to use digital sensors instead of human sensors. These companies train Deep Learning algorithms to use [large amounts of data](https://medium.com/breathe-publication/top-15-deep-learning-applications-that-will-rule-the-world-in-2018-and-beyond-7c6130c43b01).
+
+2. Natural Language Processing
+
+Machines are taught to understand complexities associated with languages such as semantics. To achieve this, [NLP through Deep Learning](https://www.mygreatlearning.com/blog/natural-language-processing-tutorial/) plays a significant role. NLP also catch linguistic nuances and frame appropriate responses.
+
+3. Healthcare
+
+Deep Learning is completely revolutionizing the healthcare and the medical industries. According to [Medium](https://medium.com/breathe-publication/top-15-deep-learning-applications-that-will-rule-the-world-in-2018-and-beyond-7c6130c43b01), AI has enabled healthcare and medical industries to advance tremendously.
```

Comment:
I don't think Medium is the author of the article. Kindly change the sentence according to the author's/ organization's name. According to "author's/organization's name"